data/_models/shared/Processes.csv

```diff
@@ -1,119 +1,143 @@
+Metabolomic mass spec assays,,ionization source,string,,,,,,,,,http://purl.obolibrary.org/obo/CHMO_0000960,A piece of apparatus that takes an analyte and generates ions which are passed to the rest of a mass spectrometer for analysis.,FAIR Genomes metabolomics add-on
+Metabolomic mass spec assays,,chromatography column,string,,,,,,,,,http://purl.obolibrary.org/obo/CHMO_0000997,A piece of apparatus that has the form of a tube with a diameter between 5 to 50 mm and hosts the stationary bed in chromatography.,FAIR Genomes metabolomics add-on
+Metabolomic mass spec assays,,phase separation protocol,text,,,,,,,,,http://purl.obolibrary.org/obo/CHMO_0002214,Any method for the synthesis of polymer membranes that involves the polymer transforming from the liquid phase of the cast solution to the solid phase.,FAIR Genomes metabolomics add-on
+Metabolomic mass spec assays,,source conditions,string,,,,,,,,,http://purl.obolibrary.org/obo/ZECO_0000104,Condition in which variables of the experiment are manipulated to determine the variables effect by comparing to a control condition.,FAIR Genomes metabolomics add-on
+Metabolomic analyses,Processes,,,,,,,,,,,http://edamontology.org/data_0949,"Basic information, annotation or documentation concerning a metabolomics analysis workflow (but not the workflow itself).",FAIR Genomes metabolomics add-on
+Metabolomic analyses,,analysis type,ontology,,,CatalogueOntologies,Metabolomic analysis type,,,,,http://edamontology.org/operation_2945,Apply analytical methods to existing data of a specific type.,FAIR Genomes metabolomics add-on
+Metabolomic analyses,,workflow definition,text,,,,,,,,,http://semanticscience.org/resource/SIO_000127,"File content or location that encapsulates the analysis workflow, i.e. a sequence of operations to achieve one or more objectives.",FAIR Genomes metabolomics add-on
+Metabolomic analyses,,workflow engine,string,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C48293,"The engine used to execute the workflow such as Papermill, Jupyter, Galaxy or Nextflow.",FAIR Genomes metabolomics add-on
+Metabolomic analyses,,workflow published,string,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C48471,A printed or electronic work offered for distribution.,FAIR Genomes metabolomics add-on
+Metabolomic analyses,,containerized by,string,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C43186,The analysis has been boxed by a container engine such as Singularity or Docker.,FAIR Genomes metabolomics add-on
+Metabolomic analyses,,software tools,string,,,,,,,,,http://purl.obolibrary.org/obo/ERO_0000071,A general term primarily used for digitally stored data such as computer programs and other kinds of information read and written by computers.,FAIR Genomes metabolomics add-on
```

Comment:
ah, that is interesting. This we also expect in the Sequencing run (pipeline used for the variant calling) and Variant interpretation analysis  (pipeline or tool used for annotation/filtering, which ideally should be documented on the files used???)